make/package-kube

```diff
@@ -9,7 +9,7 @@ ARTIFACT=uaa-kube-${GIT_DESCRIBE}.zip
 
 rm ${GIT_ROOT}/${ARTIFACT} 2>/dev/null || true
 
-cd kube && zip -r9 ${GIT_ROOT}/${ARTIFACT} * \
+cd kube && zip -r9 ${GIT_ROOT}/${ARTIFACT} * ../kube-test/* \
```

Comment:
The last commit looks quite good to me. The way it is done it actually also ensures that the SCF kube-dist picks up the new file without having to be changed at all. Very nice.